### ISSUE TITLE
ErrorExceededFindCountLimit(-8) / (Status: 804a0005)

### ISSUE THREAD

**Issue by ermingol** [Ericsson/exchangecalendar#374]
I just tried to decline an invite to a meeting. This resulted in this dialog
![screenshot from 2015-10-30 11 30 55](https://cloud.githubusercontent.com/assets/11158653/10843539/e2e4c90a-7ef9-11e5-8bb8-07ee91f89674.png)
and then I get this status message for the meeting
![screenshot from 2015-10-30 11 31 19](https://cloud.githubusercontent.com/assets/11158653/10843550/f5288200-7ef9-11e5-9a4c-1fba139341eb.png)
and if I select another mail and then the meeting invite again I get this status message
![screenshot from 2015-10-30 11 33 43](https://cloud.githubusercontent.com/assets/11158653/10843578/220d555c-7efa-11e5-931c-ceb034d58fe3.png)

My calendar is set to writable in the preferences.

**Comment by bavincen** [comment]
Your calendar become read-only or event  already got deleted

**Comment by bavincen** [comment]
Hi @ermingol 
is that a recurring event.

Yes calendar events is not available in said calendar if that is not readonly

**Comment by bavincen** [comment]
Are you getting the above error sametime.

**Comment by ermingol** [comment]
I get the "Error on..." dialog when I press the "decline" button. It is not a recurring event. Then immediately I get the "Status: 804a0005" in the row where the "decline" button was.

When I view the properties for the calendar the "read only" checkbox is **not** ticked.

**Comment by bavincen** [comment]
Your Calendar, Lightning, TB version please?

**Comment by ermingol** [comment]
Calendar: 3.5.0-beta1
Lightning: 4.0.3
TB: 38.3.0

**Comment by ermingol** [comment]
I get this error when I also try to accept a meeting invitation. Regardless of it is a recurring event or not. Since I reported the initial issue I have restarted TB several times (and also rebooted my Linux machine several times).

Apart from this, I can view all meetings and snooze/dismiss them without problems. But not being able to accept/decline starts getting a bit annoying. :(

Please let me know if I can assist in some way, e.g providing logs or whatever.

**Comment by ermingol** [comment]
I have updated to 3.6.0-beta1 (which contains the fix for #417) and I still get this error message when I try to accept a meeting invitation from within the mail view. :(

When still on 3.5.0-beta1 I found that I could accept meeting invitations from within the calendar view (double-click on a meeting and choose "I will attend"), and this workaround still works in 3.6.0-beta1.

**Comment by ermingol** [comment]
I do not know if this matters, but I have currently 2114 unread messages in my mailbox and in total 11452 messages according to Thunderbird
![screenshot from 2016-03-09 09 27 49](https://cloud.githubusercontent.com/assets/11158653/13629409/42725fd6-e5d9-11e5-9d47-d1d92fd35e75.png)
I can also successfully accept meeting invitations if I click on the orange "Invitations: 8" link in the image above.

**Comment by ermingol** [comment]
Sorry. Pressed wrong button. :(

**Comment by bavincen** [comment]
This one was fixed.. can you use recent plugin..

**Comment by bavincen** [comment]
@Trim  one for you! :)

**Comment by ermingol** [comment]
> This one was fixed.. can you use recent plugin..

I have this version of the plugin installed
![screenshot from 2016-03-09 15 20 50](https://cloud.githubusercontent.com/assets/11158653/13638030/aa94c83e-e60a-11e5-863a-d93022041346.png)

Can you please tell me where I can find a newer version to try?

**Comment by ermingol** [comment]
Also, can you please tell me if I need to do something special before installing? I just installed 3.6.0-beta1 on top of 3.5.0-beta1...

**Comment by Trim** [comment]
I'm a bit surprised that you reach a FindItem limit with only 8 meetings, but maybe one of these meetings is a recurring event.

I've found [some Microsoft example](https://msdn.microsoft.com/en-us/library/office/dn495614%28v=exchg.150%29.aspx#code-snippet-4) showing us how to find a Meeting with a CalendarView (which allow to be more robust against recurring events).

The issue with such solution is that the design of the calling function will need many modifications.
I'll check what is possible to be done easily.

Maybe another solution is to ask only Id for meetings and get details only in a second step (as done in the Microsoft above example).

**Comment by ermingol** [comment]
Today I have 11 unread meetings, i.e.
![screenshot from 2016-03-10 08 18 01](https://cloud.githubusercontent.com/assets/11158653/13662163/c98edea8-e698-11e5-8531-8f4ac39f2277.png)
None of these 11 meetings are recurring. Still I can't accept any of those meeting invitations directly from the mail view (get the same error)...

But I do have many more (accepted and unanswered) meetings (some of them recurring) in my calendar. And now I also see that I do have recurring meetings that I have not answered yet that are not included in the list given when clicking on the orange "Invitations: NN" link above. And these are also not included in the presented "NN" total number.

**Comment by ermingol** [comment]
I have also noticed that when clicking on the "Invitations" at the bottom of the window (to get a list of all invitations) I can accept invitations via that windows. But if I try via the mail window I get the error message...

**Comment by bavincen** [comment]
please check that invitation and the calenadar item exists, or write operation is not locked

**Comment by ermingol** [comment]
Yes, the invitation and calendar items exists and write operation is not locked.
